game-core/src/test/java/games/strategy/debug/error/reporting/ErrorReportConfigurationTest.java

```diff
@@ -0,0 +1,22 @@
+package games.strategy.debug.error.reporting;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+import org.sonatype.goodies.prefs.memory.MemoryPreferences;
+import org.triplea.test.common.Integration;
+
+import games.strategy.triplea.settings.ClientSetting;
+
+
+@Integration
+class ErrorReportConfigurationTest {
+
+  @Test
+  void verifyConstructionIsWithoutError() {
+    ClientSetting.setPreferences(new MemoryPreferences());
```

Comment:
This boilerplate can be avoided by extending `AbstractClientSettingTestCase` (which also correctly resets the `Preferences` reference after each test to avoid leaking data across tests).

Reply:
4c024707c